shared-libs/contacts/test/unit/places.spec.js

```diff
@@ -404,6 +404,20 @@ describe('places controller', () => {
       });
     });
 
+    it('returns err if contact does not exist', () => {
+      const place = {
+        name: 'HC',
+        type: 'district_hospital',
+        contact: 'person'
+      };
+      fetchHydratedDoc.rejects({ status: 404 });
+      const post = db.medic.post;
+      return controller._createPlaces(place).catch(err => {
+        chai.expect(err.message).to.equal('Failed to find person.');
+        chai.expect(post.callCount).to.equal(0);
```

Comment:
Unfortunately, we need more there. Adding assertions in the `catch` _only_ works if you also add a throw if the call is successful. Otherwise, if the call resolves, your test passes and you don't run any assertion: 
```suggestion
try {
  await controller._createPlaces(place);
  chai.expect.fail('Call should throw');
} catch (err) {
  chai.expect(err.message).to.equal('Failed to find person.');
  chai.expect(post.callCount).to.equal(0);
}
```

I'm noticing the same pattern in other unit tests as well. Those should be changed too!